### PR TITLE
Tech CORE-400 | Revaluate nomenclature of scope

### PR DESCRIPTION
**Pull request by franciscobonand**
# Description

Kind: Tech Pendency / Refactor

Section: `/pkg` and `/cmd`

Summary: Rename "context" to "scope" when referring to Inspr structures / Inspr tree

## Changelog

> Renamed comments, func params and variables from "context" to "scope"
> Refactored some error messages
> Renamed env var "INSPR_APP_CTX" to "INSPR_APP_SCOPE"
> Moved "InitDO" to models file in `pkg/auth`

## Pay attention to

> If some of the renamings doesn't make sense

### CODE REVIEW

**Review by nicholaspcr — APPROVED** [review]

File: examples/k8s_op_demo/testpod.yaml [hunk 1]
```diff
@@ -8,24 +8,23 @@ spec:
...
+        - name: INSPR_APP_ID
+          value: "abcdef"
+        - name: "NODES_APPS_NAMESPACE"
+          value: "default"
+        - name: "INSPR_INPUT_CHANNELS"
+          value: "default"
+        - name: "INSPR_OUTPUT_CHANNELS"
+          value: "default"
+        - name: "INSPR_UNIX_SOCKET"
+          value: "default"
```
Comment (LucasStarlingdePaulaSalles) [inline examples/k8s_op_demo/testpod.yaml:30]: This indentation change is intended?

File: examples/sidecar_demo/testpod.yaml [hunk 2]
```diff
@@ -12,8 +12,8 @@ spec:
       image: gcr.io/red-inspr/inspr/sidecar/kafka:0b6a876-dirty
       imagePullPolicy: Always
       volumeMounts:
-      - name: inspr-volume
-        mountPath: /inspr
+        - name: inspr-volume
+          mountPath: /inspr
```
Comment (LucasStarlingdePaulaSalles) [inline examples/sidecar_demo/testpod.yaml:16]: This indentation change is intended?

File: examples/sidecar_demo/testpod.yaml [hunk 3]
```diff
@@ -41,28 +41,28 @@ spec:
       image: gcr.io/red-inspr/inspr/sidecar/test:latest
       imagePullPolicy: Always
       volumeMounts:
-      - name: inspr-volume
-        mountPath: /inspr
+        - name: inspr-volume
+          mountPath: /inspr
```
Comment (LucasStarlingdePaulaSalles) [inline examples/sidecar_demo/testpod.yaml:45]: This indentation change is intended?

File: examples/sidecar_demo/testpod.yaml [hunk 4]
```diff
@@ -41,28 +41,28 @@ spec:
...
+        - name: KAFKA_BOOTSTRAP_SERVERS
+          value: "default.kafka.svc:9092"
+        - name: KAFKA_AUTO_OFFSET_RESET
+          value: "earliest"
+        - name: ch1_SCHEMA
+          value: '{"type":"string"}'
+        - name: INSPR_SIDECAR_IMAGE
+          value: ""
+        - name: INSPR_APP_ID
+          value: "abcdef"
```
Comment (LucasStarlingdePaulaSalles) [inline examples/sidecar_demo/testpod.yaml:68]: This indentation change is intended?

**Review by LucasStarlingdePaulaSalles — APPROVED** [review]

File: examples/k8s_op_demo/testpod.yaml (hunk 1 above)
Comment (franciscobonand) [reply examples/k8s_op_demo/testpod.yaml:30]: VSCode probably changed it automatically

**Pull request merged**